pkg/webui/account/containers/tokens-table/tokens-table.js

```diff
@@ -0,0 +1,201 @@
+// limitations under the License.
+
+import React from 'react'
+import { defineMessages } from 'react-intl'
+import { Col, Row, Container } from 'react-grid-system'
+
+import toast from '@ttn-lw/components/toast'
+import Button from '@ttn-lw/components/button'
+import SafeInspector from '@ttn-lw/components/safe-inspector'
+import Breadcrumb from '@ttn-lw/components/breadcrumbs/breadcrumb'
+import { useBreadcrumbs } from '@ttn-lw/components/breadcrumbs/context'
+
+import FetchTable from '@ttn-lw/containers/fetch-table'
+
+import DateTime from '@ttn-lw/lib/components/date-time'
+
+import PropTypes from '@ttn-lw/lib/prop-types'
+import sharedMessages from '@ttn-lw/lib/shared-messages'
+
+import { getAuthorizationTokensList } from '@account/store/actions/authorizations'
+
+import {
+  selectTokens,
+  selectTokensTotalCount,
+  selectTokensFetching,
+} from '@account/store/selectors/authorizations'
+
+const m = defineMessages({
+  deleteSuccess: 'Access token invalidated',
+  deleteFail: 'There was an error and the access token could not be invalidated',
+  deleteButton: 'Invalidate this access token',
+  deleteAllSuccess: 'All access tokens invalidated',
+  deleteAllFail: 'There was an error and the access tokens could not be invalidated',
+  deleteAllButton: 'Invalidate all access tokens',
+  expires: 'Expires',
+})
+
+const TokensTable = props => {
+  const { userId, clientId, deleteToken, tokenIds, deleteAllTokens, navigateToList, ...rest } =
+    props
+
+  useBreadcrumbs(
+    'client-authorizations.single.tokens',
+    <Breadcrumb path={`/client-authorizations/${clientId}/tokens`} content={'tokens'} />,
+  )
+
+  const handleDeleteToken = React.useCallback(
+    async id => {
+      try {
+        await deleteToken(id)
+        toast({
+          title: clientId,
+          message: m.deleteSuccess,
+          type: toast.types.SUCCESS,
+        })
+        navigateToList(clientId)
+      } catch (err) {
+        toast({
+          title: clientId,
+          message: m.deleteFail,
+          type: toast.types.ERROR,
+        })
+      }
+    },
+    [deleteToken, clientId, navigateToList],
+  )
+
+  const handleDeleteAllTokens = React.useCallback(async () => {
+    try {
+      await deleteAllTokens(tokenIds)
+      toast({
+        title: clientId,
+        message: m.deleteAllSuccess,
+        type: toast.types.SUCCESS,
+      })
+      navigateToList(clientId)
+    } catch (err) {
+      toast({
+        title: clientId,
+        message: m.deleteAllFail,
+        type: toast.types.ERROR,
+      })
+    }
+  }, [deleteAllTokens, clientId, tokenIds, navigateToList])
+
+  const headers = React.useMemo(() => {
+    const baseHeaders = [
+      {
+        name: 'id',
+        displayName: sharedMessages.id,
+        width: 40,
+        getValue: row => ({
+          id: row.id,
+        }),
+        render: details => (
+          <SafeInspector data={details.id} noTransform noCopyPopup small hideable={false} />
```

Comment:
```suggestion
          <SafeInspector data={details.id} noTransform noCopyPopup small hideable={false} isBytes={false} />
```
Otherwise the data will be displayed in pairs of two characters.